app/controllers/users_controller.rb

```diff
@@ -27,9 +33,40 @@ def create
         redirect_to new_user_path
       end
     end
+
+    # PATCH/PUT /users/1 or /users/1.json
+    def update
+      @user = User.find(params[:id])
+
+      respond_to do |format|
+        if @user.update(user_params)
+          format.html { redirect_to user_url(@user), notice: "User was successfully updated." }
+          format.json { render :show, status: :ok, location: @user }
+        else
+          format.html { render :edit, status: :unprocessable_entity }
+          format.json { render json: @user.errors, status: :unprocessable_entity }
+        end
+      end
+    end
+
+    # DELETE /people/1 or /people/1.json
+  def destroy
+    @user.destroy
+
+    respond_to do |format|
+      format.html { redirect_to people_url, notice: "User was successfully destroyed." }
+      format.json { head :no_content }
+    end
+  end
   
     private 
     def user_params
-      params.require(:user).permit(:email, :password)
```

Comment:
can add password back in later, just not needed at the moment; really :group_id is the only parameter where implementation is added